pages/vi/vi-first-steps.md

```diff
@@ -8,6 +8,8 @@ If you are selected for the internship after completing the steps, you will be o
 
 Once accepted, you and your team will work on an assignment, and we’ll switch up the assignments each week. As part of this internship, you will have the opportunity to work with software and languages including **[Git](https://git-scm.com/)**, **[GitHub](https://github.com/)**, **[Markdown](https://daringfireball.net/projects/markdown/syntax)**, **[Vagrant](https://www.vagrantup.com/)**, **[VirtualBox](https://www.virtualbox.org/)**, **[Command Line/Terminal](https://en.wikipedia.org/wiki/Command-line_interface)**, **[Command Line/Terminal Scripts](https://www.codecademy.com/articles/command-line-commands)**, **[Vim](https://www.vim.org/)**, **[CouchDB](http://couchdb.apache.org/)**, **[Docker](https://www.docker.com/)**, **[HTML5](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/HTML5)**, **[JavaScript](https://en.wikipedia.org/wiki/JavaScript)**, **[Node.js](https://nodejs.org/en/)**, and **[Angular](https://angular.io/)**.
 
+For IDE/Editor, we recommend **[Sublime](https://www.sublimetext.com/)**, **[Visual Studio](https://code.visualstudio.com/)**, **[WebStorm](https://www.jetbrains.com/webstorm/?fromMenu)**. Sublime and Visual Studio are both light editors, but WebStorm has a lot of useful features. If you have a school email, you can use it to get a free license from **[jetbrains](https://www.jetbrains.com/)**.
```

Comment:
Visual Studio and VS code are two different editors. VS code is the lighter version not Visual Studio.